docs/src/conformal_cubed_sphere.md

```diff
@@ -54,43 +64,44 @@ colgap!(fig.layout, 40)
 current_figure()
 ```
 
-Above, we plotted the mapping from the cube's face onto the sphere both in a 2D
-projection (e.g., overlooking the sphere down to its North Pole) and in 3D space.
+Above, we plotted the mapping from the cube's face onto the sphere both in a 2D projection (e.g., overlooking the sphere down to its North Pole) and in 3D space.
 
-We can then use [Rotations.jl](https://github.com/JuliaGeometry/Rotations.jl) to rotate
-the face of the sphere that includes the North Pole and this way obtain all six faces of
-the sphere.
+We can then use [Rotations.jl](https://github.com/JuliaGeometry/Rotations.jl) to rotate the face of the sphere that includes the North Pole and this way obtain all six faces of the sphere.
 
 ```@example 1
 using Rotations
 
-fig = Figure(resolution = (800, 400))
+fig = Figure(resolution = (1500, 750))
 
-ax2D = Axis(fig[1, 1], aspect = 1)
-ax3D = Axis3(fig[1, 2], aspect = (1, 1, 1), limits = ((-1, 1), (-1, 1), (-1, 1)))
+ax2D = Axis(fig[1, 1]; aspect = 1, title = "Cubed Sphere", axis_kwargs_2D...)
+ax3D = Axis3(fig[1, 2]; aspect = (1, 1, 1), limits = ((-1, 1), (-1, 1), (-1, 1)), title = "Cubed Sphere", 
+             axis_kwargs_3D...)
 
 for ax in [ax2D, ax3D]
-    wireframe!(ax, X, Y, Z)
+    wireframe!(ax, X, Y, Z, color = colors[1], alpha = alphas[1])
```

Comment:
this line errors..

https://github.com/CliMA/CubedSphere.jl/actions/runs/7821625824/job/21338990762?pr=34#step:6:403

you can try to build the docs locally by calling:

```
rm -rf docs/build; JULIA_DEBUG=Documenter julia --color=yes --project=docs docs/make.jl
```

and then open `docs/build/index.html`.

Reply:
@navidcy thanks for this information. I didn't know how to build the docs locally.

Reply:
It’s the same process as the GitHub action. 

also see https://clima.github.io/OceananigansDocumentation/stable/contributing/#Documentation